server/routes/maintainingReports/index.ts

```diff
@@ -90,14 +92,8 @@ export default function Index(services: Services): Router {
     post('/coordinator/report/:reportId/add-staff', coordinator.submitAddInvolvedStaff)
     get('/coordinator/report/:reportId/add-staff/result/:result', coordinator.viewAddInvolvedStaffResult)
 
```

Comment:
L93 - 100 moved to L84 and L85 but with first parts of endpoint removed to match the other endpoints in edit